src/Microsoft.SqlTools.ServiceLayer/SqlContext/QueryExecutionSettings.cs

```diff
@@ -25,13 +33,13 @@ public class QueryExecutionSettings : GeneralRequestDetails
         /// <summary>
         /// Default number of chars to store for long text fields (de facto standard as per SSMS)
         /// </summary>
-        private const int DefaultMaxCharsToStore = 65535; // 64 KB - QE default
+        private const int DefaultMaxCharsToStore = int2MB; // #392: Raised from 64K to 2MB as found in SSMS
```

Comment:
we probably don't need the comment regarding the history of the value.  anyone interested where the value came from can always check the source control history.